vipps-ecom-api-quick-start.md

```diff
@@ -18,140 +18,80 @@ You can also perform express checkout payments and get access to user info.
 ## Table of contents
```

Comment:
You can delete the whole toc

Reply:
It's quite short, and I like it. 😅

Reply:
Yes, but to be difficult:  it is not providing nearly as useful information as most of the stuff you have deleted already. Especially since the file is short, it doesn't need it. 
I also liked a lot of the stuff that is being deleted 😿